subsys/bluetooth/host/gatt.c

```diff
@@ -1438,6 +1438,27 @@ static void gatt_delayed_store_enqueue(uint8_t id, const bt_addr_le_t *peer_addr
 	}
 }
 
+static void do_delayed_store(uint8_t id, const bt_addr_le_t *peer_addr)
+{
+	struct ds_peer *el = gatt_delayed_store_find(id, peer_addr);
+
+	if (bt_addr_le_is_bonded(id, peer_addr)) {
```

Comment:
I would like a local test that `el` is not NULL.

```suggestion
	if (el && bt_addr_le_is_bonded(id, peer_addr)) {
```